packages/vscode-extension/src/extension/decorators.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import * as vscode from "vscode";
+import { LanguageClient } from "vscode-languageclient/node.js";
+import { Settings } from "./settings";
+import { Range } from "vscode-languageserver-types";
```

Comment:
Suggestion: The `vscode-languageclient` package exports everything from `vscode-languageserver-types`. We don't need a specific dependency to `vscode-languageserver-types`.
```suggestion
import { LanguageClient, Range } from "vscode-languageclient/node.js";
import { Settings } from "./settings";
```